### PR TITLE
Added Apache Kafka Connect gcs sink

### LINKED ISSUE #633: Migrate Kafka GCS sink connector
# What should we add to our content collections?

Migrate https://help.aiven.io/en/articles/2888122-aiven-for-apache-kafka-gcs-sink-connector-by-aiven

# Is this something you would work on yourself?

[X] I plan to open a pull request for this

### PR DESCRIPTION
**Pull request by ftisiot**
# What changed, and why it matters

Migrated https://help.aiven.io/en/articles/2888122-aiven-for-apache-kafka-gcs-sink-connector-by-aiven

Fixes #633

### CODE REVIEW

File: docs/products/kafka/kafka-connect/howto/gcs-sink.rst [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+
+Create a Kafka Connect connector with the Aiven Console
+'''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+To create the connector, access the `Aiven Console <https://console.aiven.io/>`_ and select the Aiven for Apache Kafka® or Aiven for Apache Kafka Connect® service where the connector needs to be defined, then:
+
+1. Click on the **Connectors** tab
+2. Clink on **Create New Connector**, the button is enabled only for services :doc:`with Kafka Connect enabled <enable-connect>`.
+3. Select the **OpenSearch sink**
```
Comment (dbeech) [inline docs/products/kafka/kafka-connect/howto/gcs-sink.rst:78]: OpenSearch? should be GCS.

File: docs/products/kafka/kafka-connect/howto/gcs-sink.rst [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+5. Paste the connector configuration (stored in the ``gcs_sink.json`` file) in the form
+6. Click on **Apply**
+
+.. Note::
+
+    The Aiven Console parses the configuration file and fills the relevant UI fields. You can review the UI fields across the various tab and change them if necessary. The changes will be reflected in JSON format in the **Connector configuration** text box.
+
+7. After all the settings are correctly configured, click on **Create new connector**
+8. Verify the connector status under the **Connectors** tab
+9. Verify the presence of the data in the target OpenSearch service, the index name is equal to the Apache Kafka topic name
```
Comment (dbeech) [inline docs/products/kafka/kafka-connect/howto/gcs-sink.rst:89]: OpenSearch again. should be GCS bucket.

**Review by dbeech — CHANGES_REQUESTED** [review]
check capitalisation of "Google Cloud Storage" throughout. also quite a few copy/paste errors, references to S3 and OpenSearch in places.

**Comment by ftisiot** [comment]
Thanks @dbeech, I believe you spotted i took parts of the content from the s3 sink and some from the Opensearch one :D

**Review by dbeech — CHANGES_REQUESTED** [review]
There's still a few places on the prereq page where "Google cloud storage" needs capitalisation like "Google Cloud Storage". Other than that, all looks good.

**Comment by ftisiot** [comment]
Fixed! Thanks

File: docs/products/kafka/kafka-connect/howto/gcs-sink.rst [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+Prerequisites
+-------------
+
+To setup the Google Cloud Storage sink connector by Aiven, you need an Aiven for Apache Kafka® service :doc:`with Apache Kafka Connect enabled <enable-connect>` or a :ref:`dedicated Aiven for Apache Kafka Connect cluster <apache_kafka_connect_dedicated_cluster>`.
+
+Furthermore you need to follow the steps :doc:`to prepare the GCP account and GCS sink <gcs-sink-prereq>` and collect the following information about the target GCS bucket upfront:
+
+* ``GCS_NAME``: The name of the GCS bucket
+* ``GCS_CREDENTIALS``: The Google service account JSON service key :ref:`created during the prerequisit phase <gcs-sink-connector-google-account>`
```
Comment (dbeech) [inline docs/products/kafka/kafka-connect/howto/gcs-sink.rst:16]: typo: "prerequisit" missing the "e". (spotted by @kevinbowman-aiven earlier). 
sorry @ftisiot, but I think that's the last one!

**Review by dbeech — CHANGES_REQUESTED** [review]

File: docs/products/kafka/kafka-connect/howto/gcs-sink.rst (hunk 3 above)
Comment (ftisiot) [reply docs/products/kafka/kafka-connect/howto/gcs-sink.rst]: Fixed, thanks

**Comment by lornajane** [comment]
@TibsAtWork Could you take a look at this one please? There's some sentence case/title warnings going on. The content is good to go once you're happy (thanks @ftisiot , thanks @dbeech !)

**Comment by TibsAtWork** [comment]
I've updated the vale styles to cope with Google Cloud Storage correctly, and fixed the errors produced by `make spell`. All seems well now.

**Review by TibsAtWork — APPROVED** [review]
Previous reviewers happy with the content, and the linter is also now happy.

**Pull request merged**